application-admintools-default/src/main/resources/templates/configurationTemplate.vm

```diff
@@ -80,4 +82,33 @@
   #warning($warningMessage)
 #end
 
+<div class="modal fade" id="viewLogsModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span></button>
+        <h1 class="modal-title" id="myModalLabel" style="text-align: center;">
+          $escapetool.xml($services.localization.render('adminTools.dashboard.modal.logs.title'))</h1>
+      </div>
+      <div class="modal-body" style="text-align: center;">
+        <form id="downloadForm" action="$services.admintools.getContextPath()/rest/admintools/files/logs"
+          method="POST" target="_blank">
+          <br>
+          <label>$escapetool.xml($services.localization.render('adminTools.dashboard.modal.logs.selection')) </label>
+          <input type="number" id="noLines" name="noLines" placeholder="100">
```

Comment:
These should be part of a `dl` tag. Check https://www.xwiki.org/xwiki/bin/view/Documentation/DevGuide/FrontendResources/VerticalForms/